src/BubbleSort.java

```diff
@@ -0,0 +1,20 @@
+import java.util.Arrays;
+
+public class BubbleSort {
+
+        public static void main(String[] args) {
+        // write your code here
+        int[] arr = {5, 2, 1, 10, 50, 10};
```

Comment:
Please rename the `int[] arr` into something more readable. 
I.E. `arrayToSort`

Reply:
done